Styleguide.md

```diff
@@ -350,170 +351,154 @@ export default es6;
 
 ## [Variables](#variables)
 
-Group all your `const`s and then group all your `let`s.
+Group all your const declarations together and then group all your let declarations together for better readability.
 
 ```js
 // bad
-let i, len, dragonball,
-    items = getItems(),
-    goSportsTeam = true;
+let index,
+  total,
+  projectName,
+  data = fetchData(),
+  isActive = true;
 
 // bad
-let i;
-const items = getItems();
-let dragonball;
-const goSportsTeam = true;
-let len;
+let index;
+const data = fetchData();
+let projectName;
+const isActive = true;
+let total;
 
 // good
-const goSportsTeam = true;
-const items = getItems();
-let dragonball;
-let i;
-let length;
+const isActive = true;
+const data = fetchData();
+let projectName;
+let index;
+let total;
 ```
 
-Assign variables where you need them, but place them in a reasonable place.
+## [Trailing commas](#commas)
+
+Using an additional trailing comma in objects, arrays, and function parameters can lead to cleaner git diffs and improve code maintenance. Here's why:
+
+1. Cleaner Git Diffs: Adding a trailing comma minimizes the number of lines changed when new elements are added, making it easier to review code changes.
+2. Consistency: Ensures a consistent style in your codebase, which can improve readability.
+3. Transpiler Support: Transpilers like Babel will remove trailing commas in the transpiled code, ensuring compatibility with older browsers.
+
+**Example: Objects**
 
 ```js
-// bad - unnecessary function call
-function checkName(hasName) {
-  const name = getName();
+// Bad - without trailing comma
+const hero = {
+  firstName: "Florence",
+  lastName: "Nightingale"
+};
 
-  if (hasName === 'test') {
-    return false;
-  }
+// Good - with trailing comma
+const hero = {
+  firstName: "Florence",
+  lastName: "Nightingale",
+};
+```
 
-  if (name === 'test') {
-    this.setName('');
-    return false;
-  }
+**Example: Arrays**
 
-  return name;
-}
+```js
+// bad
+const heroes = ["Batman", "Superman"];
 
 // good
-function checkName(hasName) {
-  if (hasName === 'test') {
-    return false;
-  }
+const heroes = ["Batman", "Superman"];
+```
 
-  const name = getName();
```

Comment:
Did you remove this in the end? Regarding the ordering, I guess I agree, but it would depend on the `getName` call complexity if it should be done before the if or just as a let before, and the calculation if needed.

Reply:
Do you want this example returned and refactored? @DarkoKukovec